src/read/elf/section.rs

```diff
@@ -873,6 +875,25 @@ pub trait SectionHeader: Debug + Pod {
         Ok(Some(relrs))
     }
 
+    /// Return the `Elf::Crel` entries in the section.
+    ///
+    /// Returns `Ok(None)` if the section does not contain relative relocations.
+    /// Returns `Err` for invalid values.
+    fn crel<'data, R: ReadRef<'data>>(
+        &self,
+        endian: Self::Endian,
+        data: R,
+    ) -> read::Result<Option<(CrelIterator<'data>, SectionIndex)>> {
+        if self.sh_type(endian) != elf::SHT_CREL {
+            return Ok(None);
+        }
+        let data = self
+            .data_as_array(endian, data)
```

Comment:
```suggestion
            .data(endian, data)
```
Won't affect correctness but slightly clearer what is happening.